doc/i2p.md

```diff
@@ -0,0 +1,64 @@
+# I2P SUPPORT IN BITCOIN
```

Comment:
No need to use full-caps here. We don't use this in other documents, and besides, the idea of a high-level formatting language like markdown is to leave it up to the rendering engine how different styles are shown.

Reply:
I actually lowercased that **before** reading your comment! Also changed BITCOIN to Bitcoin Core, since this doc is specific to Bitcoin Core.

It was a copy-paste-modify from doc/tor.md which uses uppercase.